desc/configuration.py

```diff
@@ -1449,7 +1449,14 @@ def body_fun(k_rhok_thetak_Rk_Zk):
 
         return jnp.vstack([rho, theta, phi]).T
 
-    def is_nested(self, nsurfs=10, ntheta=20, zeta=0, Nt=45, Nr=20):
+    def is_nested(
+        self,
+        nsurfs=10,
+        ntheta=20,
+        zetas=np.linspace(0, 2 * np.pi, 4, endpoint=False),
```

Comment:
might make more sense to make this a scalar `nzeta` to be consistent with the others. What I've done for others is have it default to `None` and that gets treated as "zeta=0 for N=0, else zeta=range..."